exporters/prometheus/src/test/java/io/opentelemetry/exporter/prometheus/LabelNameSanitizerTest.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.exporter.prometheus;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.util.concurrent.atomic.AtomicInteger;
+import java.util.function.Function;
+import org.junit.jupiter.api.Test;
+
+class LabelNameSanitizerTest {
+
+  @Test
+  void testSanitizerCaching() {
+    AtomicInteger count = new AtomicInteger();
+    Function<String, String> delegate = labelName -> labelName + count.incrementAndGet();
+    LabelNameSanitizer testClass = new LabelNameSanitizer(delegate);
```

Comment:
```suggestion
    LabelNameSanitizer sanitizer = new LabelNameSanitizer(delegate);
```

makes the test easier to read

Reply:
Sure thing.  We can talk about the pros/cons of each approach sometime, but I'm happy to change it here.